docs/source/run.rst

```diff
@@ -64,6 +64,10 @@ Commonly Used Arguments
   adjust the name of Gunicorn process as they appear in the process system
   table (which affects tools like ``ps`` and ``top``).
 
+.. Gevent note::
```

Comment:
I know @benoitc suggested run.rst, but I'd prefer to see this in http://docs.gunicorn.org/en/latest/settings.html#worker-class